src/main/js/helpers/legend.js

```diff
@@ -70,15 +70,17 @@ const loadLegendItem = (legendSVG, t, shownTargets, eventHandlers) => {
     validateLegendLabel(t.label);
     const text = getText(t.label.display);
     const index = shownTargets.indexOf(t.key);
+    const disabled =
```

Comment:
We are force disabling the legend item here, so might consider renaming this to `shouldForceDisableLegendItem`.